### DISCUSSION TITLE
Preserve file version numbers associated with files

### DISCUSSION THREAD

**Discussion by masinter** [Interlisp/medley#61] [configurations and releases]
@nbriggs 
> [medley repo] doesn't have the version numbers associated with files which are unversioned.

**Comment by masinter** [comment]
I think the problem was also in https://github.com/Interlisp/medley/issues/49#issuecomment-715566486 
maybe Ron did a trim?

**Comment by masinter** [comment]
find . -type f -iname 'font.~*' -exec ls -l {} \;
[font-list.txt](https://github.com/Interlisp/medley/files/5469894/font-list.txt)

**Comment by nbriggs** [comment]
I think if when you paste in things with version numbers you wrap it in triple-backquotes on an otherwise empty line before and after it won't think it's markdown -- like this:
```
sample-file.~123~
```
